conf/blacklist/blacklist

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+IPTABLES=/sbin/iptables
+URL=http://feeds.dshield.org/block.txt
+FILE=/tmp/dshield_block.text
+CHAIN=dshield
+IP_TMP=/tmp/ip.tmp
+IP_BLACKLIST=/etc/ip-blacklist.conf
+IP_BLACKLIST_TMP=/tmp/ip-blacklist.tmp
+BLACKLISTS=(
+# Project Honey Pot Directory of Dictionary Attacker IPs
+"http://www.projecthoneypot.org/list_of_ips.php?t=d&rss=1" 
+# BruteForceBlocker
+"http://danger.rulez.sk/projects/bruteforceblocker/blist.php" 
+# Spamhaus
+"http://www.spamhaus.org/drop/drop.lasso" 
+# C.I. Army
+"http://cinsscore.com/list/ci-badguys.txt" 
+# OpenBL.org
+"http://www.openbl.org/lists/base.txt" 
+# Autoshun
+"http://www.autoshun.org/files/shunlist.csv"
+# Blocklist.de
+"http://lists.blocklist.de/lists/all.txt"
+# Malware Domain List
+"https://www.malwaredomainlist.com/hostslist/ip.txt"
+# ZeusTracker
+"https://zeustracker.abuse.ch/blocklist.php?download=ipblocklist"
+# malc0de IP blacklist
+"http://malc0de.com/bl/IP_Blacklist.txt"
+# MaxMind GeoIP Anonymous Proxies
+"https://www.maxmind.com/en/proxy-detection-sample-list"
```

Comment:
My comment was specifically on line 31. Why do we care about anonymous proxies? What is high-risk about that? 

Also, that list is a commercial product, it doesn't mention (at least I can't find it) a license for use.

Whenever you visit that site they use canvas tracking/web gl tracking. I don't know if I would like my servers to connect repeatedly to their machines. (It would surprise me if they fingerprinted the requests too)